### PR TITLE
MAX_SCRIPT_ELEMENT_SIZE for witness stack

### PR DESCRIPTION
**Pull request by jl2012**

### CODE REVIEW

**Comment by jl2012** [comment]
compiled but untested

File: src/script/interpreter.cpp [hunk 1]
```diff
@@ -246,6 +246,15 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
     set_error(serror, SCRIPT_ERR_UNKNOWN_ERROR);
     if (script.size() > 10000)
         return set_error(serror, SCRIPT_ERR_SCRIPT_SIZE);
+    
+    // Disallow stack item size > MAX_SCRIPT_ELEMENT_SIZE in witness stack    
+    if (sigversion == 1) {
+        for (unsigned int i = 1; i <= stack.size(); i++) {
```
Comment (sipa) [inline src/script/interpreter.cpp]: Loop should run from 0 to stack.size() - 1.

**Comment by jl2012** [comment]
tested: https://www.irccloud.com/pastebin/hHnPsUPz/

**Comment by sipa** [comment]
Merging.

**Pull request closed without merging**